xwiki-platform-core/xwiki-platform-dashboard/xwiki-platform-dashboard-ui/src/main/resources/Dashboard/SpaceDashboardTemplate.xml

```diff
@@ -210,7 +210,7 @@
       <position>1,2</position>
     </property>
     <property>
-      <title>$services.localization.render('platform.dashboard.wiki.tagcloud')</title>
+      <title>$services.localization.render('platform.dashboard.wiki.tagsgadget', [$doc.title])</title>
```

Comment:
Use $doc.plainTitle instead of $doc.title (for documents that use translation keys in their titles) from mflorea